### PR TITLE
Jia/1167 add check trading specs button in et fs

### PR DESCRIPTION
**Pull request by jia-deriv**
Changes:

-   Added check trading specs button on ETFs page
image

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/features/pages/markets/etf/check-trading/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@use 'features/styles/theme/theme-mixins' as *;
+@import 'features/styles/theme/variables';
+
+.item_container {
+    @include breakpoints(tablet, max) {
+        box-shadow: rgba(0, 0, 0, 0.1) 0 0 3px 0;
+    }
```
Comment (hasan-deriv) [inline src/features/pages/markets/etf/check-trading/styles.module.scss:7]: Do we need this? and we use mobile first approach.

**Review by mitra-deriv — APPROVED** [review]

File: src/features/pages/markets/etf/check-trading/index.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import * as styles from './styles.module.scss'
+import Flex from 'features/components/atoms/flex-box'
+import { Localize } from 'components/localization'
+import LinkButton from 'features/components/atoms/link-button'
+import Typography from 'features/components/atoms/typography'
+
+const CheckTrading = () => {
+    return (
+        <Flex.Box direction="col" className={styles.item_container}>
```
Comment (hasan-deriv) [inline src/features/pages/markets/etf/check-trading/index.tsx:10]: Container.Fixed can be used here...

**Review by mayuran-deriv — APPROVED** [review]

**Review by hasan-deriv — APPROVED** [review]

File: src/features/pages/markets/etf/check-trading/index.tsx (hunk 2 above)
Comment (jia-deriv) [reply src/features/pages/markets/etf/check-trading/index.tsx]: Alright, will add the fixed container here.

File: src/features/pages/markets/etf/check-trading/styles.module.scss (hunk 1 above)
Comment (jia-deriv) [reply src/features/pages/markets/etf/check-trading/styles.module.scss]: Following the design and copy from the styled components before atomic approach. Will change it to mobile first approach

**Review by aswathy-deriv — APPROVED** [review]

**Pull request merged**